pymysql/err.py

```diff
@@ -97,6 +97,11 @@ def _map_error(exc, *errors):
 
 def raise_mysql_exception(data):
     errno = struct.unpack('<h', data[1:3])[0]
-    errval = data[4:].decode('utf-8', 'replace')
+    is_41 = data[3:4] == b"#"
+    if is_41:
+        # client protocol 4.1
+        errval = data[9:].decode("utf8", 'replace')
+    else:
+        errval = data[3:].decode("utf8", 'replace')
```

Comment:
Sure, I can do that quickly. I just grabbed the old code & didn't look all that hard at it ;)